tests/unit/visualise/test_generate_map_based_visualisations.py

```diff
@@ -1,3 +1,4 @@
+from importlib.resources import path
```

Comment:
I can't see where this is used, is it needed? (it's also shadowed by local variables with the same name)

Reply:
Removing it hasn't caused any issues, so I guess not!